CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - Refactored `strictRoutes` by removing query parameters into functions for symmetry
 - Refactored graph and generation related route functionality into controllers
 - Moved the routes related to downloads to the `Timetable` controller to be triggered
+- Refactored the HTTPS response generation from the model in `App/Database/CourseQueries` to the controller `App/Controllers/`
```

Comment:
I would remove the words "the model in"; the file `CourseQueries.hs` isn't strictly speaking a model in the traditional MVC sense, it's more just of a collection of functions right now.